assets/sass/components/_footer.scss

```diff
@@ -0,0 +1,163 @@
+.footer {
+  width: 100%;
+&__wrapper {
+  margin: 0;
+  padding: 0;
+  border:0;
+  background-color: #333333;
+  width: 100%;
+ }
+&__adress {
+  padding-bottom: 50px;
+  margin:0;
+  padding-top: 100px;
+  font-size: 14px;
+    width: 100%;
+}
+&__adress p {
```

Comment:
See that nesting is not right. One of the points of .scss is that we can know easily what is belong to the parent element. 
So, address part of code should look like  &__adress { p {.....} p:first-child{....} p:last-child{} }
There is no needed to write 4 times &__adress. When you wirite it once, everything goes inside. All children are inside one selector.